python/tvm/relay/op/contrib/tensorrt.py

```diff
@@ -723,8 +723,12 @@ def pad_annotate_fn(expr):  # pylint: disable=unused-variable
     if attrs.pad_mode != "constant":
         logger.info("nn.pad: pad mode is %s but must be constant.", attrs.pad_mode)
         return False
-    if float(attrs.pad_value) != 0.0:
-        logger.info("nn.pad: pad value is %f but must be 0.0.", float(attrs.pad_value))
+    if (
+        isinstance(args[1], relay.Constant)
```

Comment:
Is it possible for `args[1]` to be an `Expr`? If not, then this should be an assertion.

Reply:
It may be assumed by the condition `attrs.pad_mode == "constant" before this conditions in the current implementations.
But I added the assertion bacause args[1] can be Expr in general, and prepare for changing handling of pad_mode may be happend in the future.

Reply:
Then this is incorrect. The implication of the if-condition is "if `args[1]` is not a constant then we may return True". Even you would like to prepare for dynamic padding, it should be like
```python
assert isinstance(args[1], relay.Constant)
if (len(args[1].checked_type.shape) == 0 and args[1].data.numpy().item() != 0.0):
  # ...
```

Reply:
Certainly, that makes sense. Fixed to it so.